app/APIStart.py

```diff
@@ -8,148 +8,206 @@
+
+db = SQLAlchemy(app)
+
+
+@app.before_first_request
+def setup():
+    """Check Database if it exists with the Tables of our model"""
+    mysql_engine.execute("use {};".format(database))
+    existing_tables = mysql_engine.execute("Show Tables;")
+    existing_tables = [d[0] for d in existing_tables]
+    table = User.__tablename__
+    if table not in existing_tables:
+        UserBase.metadata.drop_all(bind=db.engine)
+        ItemBase.metadata.drop_all(bind=db.engine)
+        UserBase.metadata.create_all(bind=db.engine)
+        ItemBase.metadata.create_all(bind=db.engine)
+
+
+@app.route("/")
 def index():
-    return jsonify('This is index page')
+    return "Hello World"
+
 
-@app.route('/items/create',methods=['POST'])
+@app.route('/items/create', methods=['POST'])
 def create():
     try:
-        testdb = db()
         name = request.json['name']
         loc = request.json['location']
         desc = request.json['description']
-        items = item()
-        items.newItem(name,loc,desc)
-        testdb.insert_item_db(items)
-        
-        resp = jsonify({"Action":'Item Added Successfully'})
+        item = Item(name, loc, desc)
+        db.session.add(item)
+        db.session.commit()
+        db.session.close()
+
+        resp = jsonify({"Action": 'Item Added Successfully'})
         resp.status_code = 200
         return resp
-        
+
     except Exception as e:
-        print(e)
-        
-@app.route('/items/update',methods=['POST'])
+        resp = jsonify({"error": e.__str__()})
+        resp.status_code = 200
+        return resp
+
+
+@app.route('/items/update', methods=['POST'])
 def update():
     try:
-        testdb = db()
-        item_id =request.json['item_id'] 
+        item_id = request.json['item_id']
         name = request.json['name']
         loc = request.json['location']
         desc = request.json['description']
-        items = item()
-        items.newItem(name,loc,desc)
-        result = testdb.update_item(items,item_id)
-        if result:
-            resp = jsonify({"Action":'Item Updated Successfully'})
+        item = db.session.query(Item).filter(Item.id == item_id).one()
+        item.item_name = name
+        item.location = loc
+        item.description = desc
+        db.session.commit()
+        db.session.close()
+
+        if item:
+            resp = jsonify({"Action": 'Item Updated Successfully'})
             resp.status_code = 200
             return resp
         else:
-            resp = jsonify({"Action":'Item with Item Id {} is not found'.format(item_id)})
+            resp = jsonify({"Action": 'Item with Item Id {} is not found'.format(item_id)})
             resp.status_code = 200
             return resp
-        
+
     except Exception as e:
-        print(e)
-@app.route('/items/view',methods=['GET'])
+        resp = jsonify({"error": e.__str__()})
+        resp.status_code = 200
+        return resp
+
+
+@app.route('/items/view', methods=['GET'])
```

Comment:
Remind me this tomorrow, will discuss about restfull apis,

Reply:
Ok